ark/jsonschema/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# @arktype/jsonschema
+
+## 1.0.0
+
+### Initial Release
+
+Released the initial implementation of the package.
+
+Known limitations:
+
+- No `dependencies` support
+- No `if`/`else`/`then` support
+- `multipleOf` only supports integers
```

Comment:
@ssalbdivad are you open to lifting ArkType's restriction on divisors being non-integers? Or, if not/as an interim solution, it seems this could be added as a custom `narrow` constraint?

Reply:
You can add whatever arbitrary validation you want in a narrow, but the reason we don't support this is because there isn't a straightforward way to test float divisibility without complex + potentially fallible string math.

@TizzySaurus was actually working on an implementation but it doesn't make sense to add a dependency or implement that much custom logic for such a niche case.

For rational values like `0.1`, you could testing something like `(n: number) => (n * 10) % 1 === 0`. Outside that though, things tend to get carry so be careful.

Reply:
Yes, I looked into this and it's just not feasible to do floating point division in NodeJS:
![image](https://github.com/user-attachments/assets/32846954-1c16-4cd0-b547-477a07642545)